src/lib/holocene/select/simple-select.svelte

```diff
@@ -54,7 +54,7 @@
 
 <style lang="postcss">
   select {
-    @apply text-slate-500;
+    @apply text-primary;
```

Comment:
Might we want to add similar styling here as well?

```suggestion
    @apply text-primary border-secondary bg-transparent;
```

| Before | After |
|--|--|
|<img width="540" alt="Screenshot 2024-04-05 at 10 41 03 AM" src="https://github.com/temporalio/ui/assets/15069288/7dd8c279-8407-4836-b6cd-4c541ab9faeb">|<img width="569" alt="Screenshot 2024-04-05 at 10 42 02 AM" src="https://github.com/temporalio/ui/assets/15069288/cfe6c280-8557-4ff0-a590-6fc3733bd793">|
|<img width="577" alt="Screenshot 2024-04-05 at 10 41 17 AM" src="https://github.com/temporalio/ui/assets/15069288/25a98066-1139-4c4b-bfc6-2cb237ae121f">|<img width="577" alt="Screenshot 2024-04-05 at 10 41 38 AM" src="https://github.com/temporalio/ui/assets/15069288/9db3ebc3-e490-4660-8800-c24b10a1a02d">|